Add Qovery as a deployment option (#96)
strapi/documentation: developer-docs/latest/deployment/qovery.md

# Qovery

This is a step-by-step guide for deploying a Strapi project on [Qovery](https://www.qovery.com). 

> Qovery is a CaaS for developer and the simplest way to deploy your full-stack apps on AWS, GCP, Azure and Digital Ocean. Qovery provides access to managed PostgreSQL and free SSL. 
> Bring your Strapi project and Qovery takes care of the deployment and hosting for you.

Reviewer: ```suggestion
Qovery is a CaaS for developers to deploy their full-stack applications on AWS, GCP, Azure and Digital Ocean. Qovery provides access to managed PostgreSQL and free SSL. 
```
I suggest we soften a bit that introduction to avoid having a more promotional tone in the documentation.
